### PR TITLE
updated slide 3 with new caption

### LINKED ISSUE #1: Add collaborators to this repo
## Comment below to be added as a collaborator

We will all be using this repository today. Even though it's public right now, you will not be able to make changes until you're given the correct permissions. We've automated this process via the GitHub API. Once you comment, we will add you as a collaborator.

You'll start to receive a lot of emails. (❗) You can slow these down immediately by clicking the **Unwatch** button at the top of the page and selecting either the **Participating and @mentions** or **Ignore** option.

### PR DESCRIPTION
**Pull request by cody-carlson-learning**
TL;DR This PR updates the copy of my slide because it was out of date.

This PR resolves #1 

## Done
- made a commit
- created a branch

## To do
- [ ] merge the branch
- [ ] delete the branch afterwards

cc. @cody-carlson

### CODE REVIEW

File: _slides/03-cody-carlson-learning.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is a cool caption!!!
```
Comment (ter-wangdavi) [inline _slides/03-cody-carlson-learning.md:6]: It is indeed cool caption haha. Approved

**Review by ter-wangdavi — APPROVED** [review]
Changes approved.

File: _slides/03-cody-carlson-learning.md (hunk 1 above)
Comment (cody-carlson) [inline _slides/03-cody-carlson-learning.md:6]: actually let's add some flair

Comment (cody-carlson-learning) [reply _slides/03-cody-carlson-learning.md]: I added some flair

**Pull request merged**